content/api/commands/mount.md

```diff
@@ -589,6 +514,172 @@ it('login link should be active when url is "/login"', () => {
 </template>
 </code-group-vue2-vue3>
 
+### Vuex
+
+To use a component that uses [Vuex](https://vuex.vuejs.org/), you can create a
+`mountWithVuex` command that expose a Vuex store to your component:
+
+<code-group-vue2-vue3>
+<template #vue2>
+
+```js
+import { mount } from '@cypress/vue'
+import Vuex from 'vuex'
+import { getStore } from '../../src/plugins/store'
+
+Cypress.Commands.add('mountWithVuex', (comp, options = {}) => {
+  // Setup options object
+  options.extensions = options.extensions || {}
+  options.extensions.plugins = options.extensions.plugins || []
+
+  // Use store passed in from options, or initialize a new one
+  options.store = options.store || getStore()
+
+  // Add Vuex plugin
+  options.extensions.plugins.push(Vuex)
+
+  return mount(comp, options)
+})
+```
+
+> The `getStore` method is a factory method that initializes Vuex and creates a
```

Comment:
Instead of `>` use an `<Alert type="info"></Alert>`

Reply:
done in 7888195